### DISCUSSION TITLE
Semi truck upgrades

### DISCUSSION THREAD

**Discussion by Roar976** [someone0002/afterlife-feedback#4] [Suggestions]
Should have it where we can upgrade  the running side  of owned semi trucks   like engine brakes  nitrous etc   also where we could put underglow  on them too currently we can customize appearance and that's it  would like it so we could do semi races and  maybe use them in some events etc like blocking a road trying to knock over a armored car.   On a separate note it would be nice if semi trailer had big storage u could use like a trunk

**Comment by someone0002** [comment]
This already a thing

**Reply by Roar976** [reply]
Tried again today and the mech tools wouldn't work on the semi just told the mech no vehicles near.   They could only change cosmetics

**Comment by Roar976** [comment]
The mechanics said the mech tools wouldn't work on them they weren't able to do  engines nitrous brakes etc  just cosmetics  and no underglow.   Is there a specific spot u need to mech tool or is it  just messed up atm

Sent from Yahoo Mail on Android 
 
  On Thu, May 19, 2022 at 4:59 PM, ***@***.***> wrote:   


This already a thing
